### ISSUE TITLE
Allow ctrl-key menu commands in thumbnails, e.g. "Print…"

### ISSUE THREAD

**Issue by SvenMichaelKlose** [linuxmint/xreader#637]
### Distribution

linux min 21.3

### Package version

xreader-4.02

### Frequency

Always

### Bug description

It is not possible to print a selected thumbnail with ctrl+p unlike the main view. That's pretty nasty for keyboard users.

### Steps to reproduce

Already explained.

### Expected behavior

Open the print dialog.

**Comment by vgstef** [comment]
Yes, I've been facing the same issue. Ctrl+P should work in the side panel too.
